pegasus/sites.v3/code.org/public/prize-terms.md

```diff
@@ -0,0 +1,26 @@
+---
+title: 'CS Leaders Prize - Terms and Conditions'
+theme: responsive
+---
+
+# CS Leaders Prize - Terms and Conditions
+
+This educational campaign is meant to celebrate and promote computer science education. The CS Leaders Prize is managed solely by Code.org, a USA-based 501c3 nonprofit whose registered office is at 1501 4th Ave, Seattle, WA 98101. These terms and conditions do not constitute an agreement with DonorsChoose.
+
+Awards will be limited to public middle and high U.S. schools only. To qualify, a school representative (staff, teacher, principal) must fill out the submission form on this page. One middle school and one high school in every U.S. state and in D.C. will receive a $10,000 gift code for DonorsChoose.org, which may be used to fund a project(s) of the school’s creation through the DonorsChoose marketplace. The award is as stated and no cash or other alternatives will be offered. The awards are not transferable. All gift codes will expire on June 30, 2023, which date cannot be extended.
```

Comment:
This mentions a submission of a form on this page. If I'm understanding correctly thats actually on a different page?

Reply:
Good call, Dani. I don't see a form on http://localhost.code.org:3000/prize-terms

Reply:
Great catch! I just linked "submission form" on line 10 and line 16 to `/prize`.